joomla/joomla-cms
# Disable Chosen.js for the rule field, again

Pull Request for Issue #12843 .

### Summary of Changes
Revert the check for instance type of data-chosen, to allow to disable Chosen for field with data-chosen="true"

### Testing Instructions
Make sure that the rule field use simple `<select>` input (without Chosen stylling),
and all icons "ok/loading/error" are visible again when try to change the rule value

### Documentation Changes Required
none
